utils/responses/responses.js

```diff
@@ -79,10 +79,10 @@ const successDTOResponse = (res, status, message) => {
  * @param msg - The message you want to send to the user
  * @returns A function that takes in a response object, a status code, and a message.
  */
-const errorDTOResponse = (res, status, message) => {
+const errorDTOResponse = (res, status, msg) => {
   return res.status(status).json({
     success: false,
-    message
+    message: msg
```

Comment:
```suggestion
    message
```
Por que puedes hacer esto

Reply:
Cñ mano esa corrección ya está en main, cuando se mergee se elige aquella